<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form9</class>
 <widget class="QWidget" name="Form9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="8,3,1,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form9</receiver>
   <slot>continueClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form9</receiver>
   <slot>cancelClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>continueClick()</slot>
  <slot>cancelClick()</slot>
 </slots>
</ui>
